### PR TITLE
Front-end 1 - network & data ingestion

### PR DESCRIPTION
**Pull request by pierluca**

### CODE REVIEW

**Comment by pierluca** [comment]
@RaulinN  still needs a bit of work, but let met know if you think this is viable.
I haven't gotten around to structuring the event reducer (I had in mind to have a list of event_ids, and an event_id->event map), but the rest should make the principles somewhat obvious.

**Comment by pierluca** [comment]
1. Fix code smell detected by Sonar
2. Revert storage modification (as further refactoring is in progress)
3. Ensure logging in the incomplete functions

File: fe1-web/__tests__/network/MessageApi.test.ts [hunk 1]
```diff
@@ -85,8 +85,8 @@ function checkDataCreateLao(obj: MessageData) {
   expect(data.name).toBeString();
   expect(data.name).toBe(mockEventName);
 
-  expect(data.creation).toBeNumber();
-  expect(data.creation).toBeGreaterThan(0);
+  expect(data.creation).toBeNumberObject();
+  expect(data.creation.valueOf()).toBeGreaterThan(0);
```
Comment (RaulinN) [inline fe1-web/__tests__/network/MessageApi.test.ts:89]: I wonder how it was working before! 🤔

**Review by RaulinN — CHANGES_REQUESTED** [review]
Encountering a TypeError in tests though ^^'

File: fe1-web/__tests__/network/MessageApi.test.ts (hunk 1 above)
Comment (pierluca) [reply fe1-web/__tests__/network/MessageApi.test.ts:89]: It was working because the mock data was in the 'wrong' type :)

**Comment by pierluca** [comment]
As discussed, I'll merge this despite the TypeScript error, as it's only a side-effect of data persistence being initialized in the tests (and it probably shouldn't)

**Pull request merged**